internal/query/projection/user_metadata.go

```diff
@@ -84,6 +84,10 @@ func (p *userMetadataProjection) reduceMetadataSet(event eventstore.Event) (*han
 	}
 	return crdb.NewUpsertStatement(
 		e,
+		[]handler.Column{
+			handler.NewCol(UserMetadataColumnInstanceID, nil),
+			handler.NewCol(UserMetadataColumnUserID, nil),
```

Comment:
```suggestion
			handler.NewCol(UserMetadataColumnUserID, nil),
			handler.NewCol(UserMetadataColumnKey, e.Key),
```

Reply:
...and remove from update cols (the key was missing on the primary key... that's why we now have metadata2)

Reply:
this still has to be fixed